gqlerrors/formatted.go

```diff
@@ -6,9 +6,16 @@ import (
 	"github.com/graphql-go/graphql/language/location"
 )
 
+type ExtendedError interface {
+	error
+	ErrorExtensions() map[string]interface{}
```

Comment:
I think we should be naming this function `Extensions` to avoid extra suffix.
It is addressed in a new PR: https://github.com/graphql-go/graphql/pull/364